src/app/modules/learning/knowledge-component/learning-objects/instructional-items/video/video.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import {BehaviorSubject} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class VideoService {
```

Comment:
`VideoPlaybackMonitor` might be a better name. Otherwise `VideoPlaybackService`.